plugins/newsdownloader.koplugin/feed_config.lua

```diff
@@ -0,0 +1,8 @@
+return {
+    -- list your feeds here:
+    -- only supports http URL for now
+    -- Atom is currently not supported, only RSS
+    -- set limit to "0" means no download, "-1" no limit.
+    { "http://www.huffingtonpost.com/feeds/index.xml", limit = 2 },
```

Comment:
Agreed, although apparently easier said than done. Browsing around a little, it would seem that most sources only give you the headline in the feed.